src/models/User.js

```diff
@@ -48,7 +48,17 @@ const userSchema = new Schema({
   level: {
     type: Number, min: 0, max: 9001, default: 0,
   },
+  exp: {
+    type: Number, min: 0, max: 2147483647, default: 0,
+  },
+  pp: {
```

Comment:
Can we please change this to something else? Like just points or currency

Reply:
You want me to remove EXP, levels and pp in favor of a single currency after I've already written the *leveling* plugin?

Reply:
No... just change the pp name?